package.json

```diff
@@ -3,45 +3,55 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
+    "\n========== Vite ==========": "",
```

Comment:
Eh ? Not so fan about this, really :thinking: 
I understand what it should do, and maybe the console even looks nice with this, but it looks like a hack around the fact that JSON forbids comments (which annoys me tbh)
But I haven't tried that, I'll give it a go offline tonight and see if I can get convinced.

And I won't try to bump right before a stream, same I'll do it tonight :sweat_smile:

Reply:
Je suis pas fan non plus, mais ca peut devenir pratique quand on a trop de scripts.